R/models.R

```diff
@@ -1,3 +1,25 @@
+#' Retrieve all models available in the Mistral API
+#'
+#' @return A character vector with the models available in the Mistral API
+#'
+#' @examples
+#' models()
+#'
+#' @export
 models <- function() {
+  base_url <- "https://api.mistral.ai"
```

Comment:
I think we can factor `base_url` out, maybe call it `mistral_base_url` in `zzz.R` ?

Reply:
It was one of my next question: what's good practice to define package-wide variables ?
`zzz.R` it is !

Reply:
Maybe look at `?rlang::on_load()` but it might be overkill